applications/app/views/fragments/newGalleryBody.scala.html

```diff
@@ -0,0 +1,127 @@
+@(gallery: model.Gallery, related: model.RelatedContent, index: Int)(implicit request: RequestHeader)
+
+@import common.{Edition, LinkTo}
+@import layout.ContentWidths.GalleryMedia
+@import views.support.TrailCssClasses.toneClass
+@import views.support.{ImgSrc, RenderClasses}
+
+<div class="l-side-margins l-side-margins--media l-side-margins--new-gallery">
+
+    <article id="article" class="@RenderClasses(
+            Map("content--advertisement-feature" -> gallery.commercial.isAdvertisementFeature,
+                "paid-content--advertisement-feature" -> gallery.commercial.isAdvertisementFeature
+            ),
+            "content", "content--media", "content--gallery", "tonal", "new-gallery", "content--immersive", s"tonal--${toneClass(gallery)}"
```

Comment:
Do we need `content--immersive`? Seems to me like it just creates more work for us as we need to override its styles later on.